modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/PaginationBarTag.java

```diff
@@ -265,19 +269,28 @@ protected int processStartTag() throws Exception {
 
 		jspWriter.write("</li>");
 
-		int totalPages = (int)Math.ceil(_totalItems / _activeDelta);
+		int totalPages = (int)Math.ceil(
+			(double)_totalItems / (double)_activeDelta);
```

Comment:
I think is enough casting one of the operands

Reply:
You're probably right and I'm not against it, but I was getting different results than in the React component (I didn't try with just "one" cast though). Just out of curiosity, are casts expensive?

Reply:
They aren't. Doing one should force the other to also be cast, as JS type coercion. But it's like less idiomatic (as it would be not relying in JS type coercion).

Reply:
Thanks, great explanation